zero_bin/rpc/Cargo.toml

```
@@ -30,3 +31,8 @@ url = { workspace = true }
 compat = { workspace = true }
 zero_bin_common = { workspace = true }
 prover = { workspace = true }
+
+[build-dependencies]
+vergen = { version = "9.0.0", features = ["build", "rustc"] }
+anyhow = { workspace = true }
+cargo_metadata = "0.18.1"
```

same here, move all references to main config file and use 
```suggestion
cargo_metadata = { workspace = true }
```